draft-ietf-ppm-dap.md

```diff
@@ -1214,64 +1269,174 @@ as follows:
 [[OPEN ISSUE: consider moving the helper report ID check into
 {{input-share-validation}}]]
 
-Once the helper has processed each valid report share in
-`AggregateInitializeReq.report_shares`, the helper then creates an
-AggregateInitializeResp message to complete its initialization. This message is
-structured as follows:
+Once the Helper has processed each valid report share in, it responds with an
+`AggregationJob` (see {{aggregation-job-representation}}). The `round` field
+MUST be 0. The order of the `prepare_steps` MUST match that of the
+`report_shares` in the Leader's request (ordered by either structure's
+`report_id` field). Each report that was marked as invalid is assigned the
+`PrepareStepState` `failed`. Otherwise, the `PrepareStepState` is either
+`continued` with the output `prep_msg`, or `finished` if the VDAF preparation
+process is finished for the report share.
+
+The Helper SHOULD store each report share's current preparation state and the
+aggregation job's round.
+
+##### POST `/tasks/{task-id}/aggregation_jobs/{aggregation-job-id}` {#aggregation-job-post}
+
+In the continuation phase, the Leader drives the VDAF preparation of each share
+in the candidate report set until the underlying VDAF moves into a terminal
+state, yielding an output share for all aggregators or an error. This phase may
+involve multiple rounds of interaction depending on the underlying VDAF. Each
+round trip is initiated by the Leader making a POST request to an aggregation
+job resource.
+
+###### Leader Continuation {#aggregation-leader-continuation}
+
+The Leader begins each round of continuation with an `AggregationJob` received
```

Comment:
You're quite right that a leader could validate the helper's `AggregationJob` immediately. The reason I put this text here is so that I don't have to duplicate it at the bottom of both "Leader Initialization" and also "Leader Continuation" (in both cases, the Leader gets an `AggregationJob` from the helper and must validate it). But I think what we could do is add a section "Aggregation Job Validation" as a peer to "Input Share Preparation" and "Input Share Validation" and then refer to that from both "Leader Initialization" and "Leader Continuation".

- [ ] refactor aggregation job validation text into another section